webrender/src/render_backend.rs

```diff
@@ -424,6 +427,7 @@ pub struct RenderBackend {
     recorder: Option<Box<ApiRecordingReceiver>>,
     sampler: Option<Box<AsyncPropertySampler + Send>>,
 
+    last_scene_id: u64,
```

Comment:
nit: any specific reason for not using `usize`?

Reply:
because all our other ids are 64-bit unconditionally (less platform-specificness)